frontend/src/pages/Dashboard/data/actions.ts

```diff
@@ -108,6 +113,35 @@ export const ChangeProfileIcon = {
   }),
 };
 
+interface UpdateUserSettingsSuccessResponse {
+  user?: User;
```

Comment:
Could this possibly not have a user? I wonder if this should be non-optional?

Reply:
It should always return the current user, provided that no exceptions are thrown. But when I make it non-optional, we get errors in the saga saying that the field may have not been initialized from the call to firebaseInteractor

Reply:
Ok works for me!